### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader:NVDA is annoucing the non interactive text as clickable.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7668]
GitHubTags: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#DesktopApp;#Win32;#NVDA
**Environment Details**:
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
**Repro Steps:**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster”.
5. "Select the deployment option" screen gets open.
6. Start NVDA and navigate with the help of up and down arrow and observe the announcement.

**Actual:**
NVDA is announcing normal and non interactive text as clickable. for eg : Label version and Deployment target. NVDA is announcing clickable version  and clickable Deployment target which just a non interactive label.
**Expected:**
NVDA should not announce the non interactive item as clickable.
**User Impact:**
Screen reader User will misunderstood content and they will think there is some interactive element on the screen.

**Attachment for Reference:**

[7668_A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3717165/7668_A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
this should have been fixed now by @Charles-Gagnon 's PR https://github.com/microsoft/azuredatastudio/commit/52de2b475108918159a6dfea59efa7407663afa7

**Comment by V-sheman** [comment]
Verified the issue in Version: 1.15.0-insider (user setup)
Commit: 56695be14ac0160e7a4d23b7af939b98fdaaea45
Date: 2020-01-30T05:35:45.707Z
VS Code: 1.42.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed hence closing the issue.
[7668 _fixed.pptx](https://github.com/microsoft/azuredatastudio/files/4132729/7668._fixed.pptx)
